src/repl.zig

```diff
@@ -538,22 +531,196 @@ pub fn ReplType(comptime MessageBus: type) type {
             }
         }
 
-        const single_repl_input_max = 10 * 4 * 1024;
+        const prompt = "> ";
+
+        fn read_until_newline_or_eof_alloc(
```

Comment:
Being pedantic, the correct action would be passing a pointer to a `*std.heap.ArenaAllocator` instead of an `Allocator`, but just the naming convention is good enough.